styles.css/main.css

```diff
@@ -252,13 +285,6 @@ table {
   color: #fff;
   border-collapse: collapse;
   text-align: left;
-  display: block;
-  font-size: .08em;
-  font: italic;
-  font: bold;
-
-
-
```

Comment:
Again, make sure you're using media queries for properties that only apply at narrow sizes